src/tables.jl

```diff
@@ -32,7 +32,7 @@ function Base.getproperty(pqrow::Row, name::Symbol)
         return missing
     else
         oid = jl_result.column_oids[col]
-        return jl_result.column_funcs[col](PQValue{oid}(jl_result, row, col))  # ::_non_null_type(T) except https://github.com/JuliaData/Missings.jl/issues/80
+        return jl_result.column_funcs[col](PQValue{oid}(jl_result, row, col))
```

Comment:
`::Base.nonmissingtype(T)`?

Reply:
That was here before but @ararslan removed it, I don't know why 😕

Reply:
I think that might have been an accident

Reply:
Oh I remember. I couldn't figure out what `T` was in this case.

Reply:
It'll be `jl_result.column_types[col]` and it seems like you won't need `nonmissingtype` after all.

Reply:
Oh, okay. So is there anything to do here then?

Reply:
Yes, add `::jl_result.column_types[col]`